.github/workflows/ci.yaml

```
@@ -0,0 +1,190 @@
+
+  docker:
+    name: Build and push docker image
+    runs-on: ubuntu-latest
+    needs: [test, lint]
+    env:
+      REGISTRY: ghcr.io
+    outputs:
+      image_tag: ${{ steps.set_output.outputs.image_tag }}
+      image_tag_short: ${{ steps.set_output.outputs.image_tag_short }}
+    steps:
+    - name: Set variables
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)